portal-api/src/v1/gef/controllers/facilities.controller.js

```diff
@@ -180,6 +180,11 @@ const facilitiesFilters = (user, filters = []) => {
     };
   }
 
+  // GEF facilities only - TODO: for @Tony
```

Comment:
Should @ttbarnes add his work here for it is merged?

Reply:
not yet. this will be needed for a future ticket that @ttbarnes will be working on. for the time being, I've had to add that condition to avoid breaking the existing workflow - this until we merge the gef and bss facilities in the UI as well (currently it's only in the database where they are merged)

Reply:
Nice, yes after my current ticket i'll be merging the collections